packages/block-editor/src/layouts/flex.js

```diff
@@ -41,12 +79,71 @@ export default {
 			` }</style>
 		);
 	},
-
 	getOrientation() {
 		return 'horizontal';
 	},
-
 	getAlignments() {
 		return [];
 	},
 };
+
+function FlexLayoutJustifyContentControl( {
+	layout,
+	onChange,
+	isToolbar = false,
+} ) {
+	const { justifyContent = 'left' } = layout;
+	if ( isToolbar ) {
+		return (
+			<JustifyContentControl
+				allowedControls={ [
+					'left',
+					'center',
+					'right',
+					'space-between',
+				] }
+				value={ justifyContent }
+				onChange={ ( value ) => {
+					onChange( {
+						...layout,
+						justifyContent: value,
+					} );
+				} }
+				popoverProps={ {
+					position: 'bottom right',
+					isAlternate: true,
+				} }
+			/>
+		);
+	}
+	return (
+		<ToggleGroupControl
+			label={ __( 'Justify content' ) }
```

Comment:
The label seems too close to the buttons in this case,
Also clicking the label doesn't seem to focus the buttons like labels do in general in controls.

I believe these are probably independent of this PR but could use some improvements separately.